howto/how-to-set-up-a-blaster.md

```diff
@@ -0,0 +1,228 @@
+const unsigned int maxLedsPerStrip = 144;
+#define CLASH_THRESHOLD_G 2.0         //Affects the shaking for unjamming your weapon.
+#define VOLUME 1500                   // 0-3000 usual range. Don't overload your speaker.
+
+// Stock required defines
+#define ENABLE_AUDIO
+#define ENABLE_MOTION
+#define ENABLE_WS2811
+#define ENABLE_SD
+
+//OLED Specific
+//#define ENABLE_SSD1306		            //To enable stock 128x32 monochrome SSD1306 OLED
+//#define INCLUDE_SSD1306               //To enable a different aspect ratio, needs some further configuration.
+
+//Blaster specific.
+#define ENABLE_BLASTER_AUTO          // If you desire to enable AUTO mode.
+#define BLASTER_SHOTS_UNTIL_EMPTY 15 // The capacity of your weapons rounds.
+#define BLASTER_JAM_PERCENTAGE 1     //  0-100. If not defined, random.
+
+
+#endif
+
+/****** CONFIG_PROP blaster defines  ****/
+#ifdef CONFIG_PROP
+
+#include "../props/blaster.h"
+
+#endif
+
+/****** CONFIG_PROP blaster defines  ****/
+#ifdef CONFIG_PRESETS
+
+Preset presets[] = {
+	// Default basic blast color with red audio flicker on blast
+	{ "blaster/E-11",
+		"tracks/mars.wav",
+		StylePtr<Lockup<BlastFadeout<BlastFadeout<Black,AudioFlicker<Black,Red>,250,EFFECT_FIRE>,AudioFlicker<Black,Blue>,1500,EFFECT_STUN>,AudioFlicker<Black,Red>>>(),
+		StylePtr<Lockup<BlastFadeout<BlastFadeout<Black,AudioFlicker<Black,Red>,250,EFFECT_FIRE>,AudioFlicker<Black,Red>,1500,EFFECT_STUN>,AudioFlicker<Black,Red>>>(),
+	  "E-11"
+	},
+  // Different prop with audio flicker and a simple color for the accent.
+	{ "blaster/E-11D",
+		"blaster/E-11D/tracks/EzraTheme.wav",
+		StylePtr<Layers<AudioFlicker<RotateColorsX<Variation,Blue>,RotateColorsX<Variation,Rgb<0,0,128>>>,BlastL<White>, InOutTrL<TrInstant,TrFade<300>,Pulsing<RotateColorsX<Variation,Blue>,RotateColorsX<Variation,Rgb<0,0,10>>,3000>>>>(),
+		StylePtr<Lockup<BlastFadeout<BlastFadeout<Black,AudioFlicker<Black,Red>,250,EFFECT_FIRE>,AudioFlicker<Black,Red>,1500,EFFECT_STUN>,AudioFlicker<Black,Red>>>(),
+		"E-11D"
+	}
+};
+
+BladeConfig blades[] = {
+ { 10000,
+   WS281XBladePtr<108, bladePin, Color8::GRB, PowerPINS<bladePowerPin2> >(),
+   SimpleBladePtr<CreeXPE2RedTemplate<1000>, NoLED, NoLED, NoLED, bladePowerPin4, -1, -1, -1>(),
+   CONFIGARRAY(presets) },
+};
+
+#endif
+
+#ifdef CONFIG_BUTTONS
+Button FireButton(BUTTON_FIRE, powerButtonPin, "fire");
+Button ModeButton(BUTTON_MODE_SELECT, auxPin, "modeselect");
+//Button PowerButton(BUTTON_POWER, aux2Pin, "power"); //A third button to power on/off your weapon
+//Button ClipButton(BUTTON_CLIP_DETECT, blade4Pin, "clip"); //Actually clip sensor. It should be closed when the clip is in, and open when the clip is removed. So you can use either latching as single button or momentary if it is a physical clip inserted sensor.
+//Button ReloadButton(BUTTON_RELOAD, blade3Pin, "reload"); //Dedicated button for reloading.
+
+#endif
+
+/*** OS 7+
+#ifdef CONFIG_STYLES
```

Comment:
Let's leave this example as basic as possible, so let's not have a CONFIG_STYLES section.,